src/SSHDebugPS/Docker/DockerHelper.cs

```diff
@@ -102,6 +103,38 @@ public static IEnumerable<DockerContainerInstance> GetLocalDockerContainers(stri
             }
         }
 
+        /// <summary>
+        /// Checks if the specified container is in the list of containers from the target host.
+        /// </summary>
+        // Another fallback option would be to: docker inspect <containerName> --format {{.State.Status}} which should return "running"
+        internal static bool IsContainerRunning(string hostName, string containerName, Connection remoteConnection)
+        {
+            IEnumerable<DockerContainerInstance> containers;
+            if (remoteConnection != null)
+            {
+                containers = GetRemoteDockerContainers(remoteConnection, hostName);
+            }
+            else
+            {
+                containers = GetLocalDockerContainers(hostName);
+            }
+
+            if (containers != null)
+            {
+                // Check if the user entered the containerName or possibly part of the Id. 
+                if (containers.Any(container => string.Equals(container.Name, containerName, StringComparison.Ordinal) ? true
```

Comment:
I think this would be logically equivalent if you just had the `.StartsWith` condition, since any string "starts with" itself.  Also, what happens if the prefix you enter matches more than one container?

Reply:
The first comparison is against the name, which has to be fully specified. the second (startswith) is against the id which is partial. 

Good point on the duplicate but then i want `docker` to do what it defaults to when there are more than 1?

Reply:
Ah, I missed that they were checking separate properties.  Sorry!